### DISCUSSION TITLE
5. Estructura de código

### DISCUSSION THREAD

**Discussion by AleRamRey** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#4] [Q&A] [answered]
Solo comentar que aun no se visualiza la estructura de código:

[Estructura y limpieza de código](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/blob/main/01%20-%20INTRO/temario/5.-estructuraCodigo.md)

**Comment by CarIosLopez** [comment]
Si aún no se visualiza es porque no esta disponible, hay que esperar a la transmisión de las 8, seguramente avisa cuando subirá lo que falta :)

**Reply by YomairaBarrera** [reply]
es a las 7 pm la transmisión compañero, todas las transmisiones del MC Rodrigo van a ser a las 7 :)

**Reply by ElizaFlores26** [reply]
hola compañeros, las transmisiones serán igual los días lunes, miércoles y viernes?

**Reply by Hisahito** [reply]
> hola compañeros, las transmisiones serán igual los días lunes, miércoles y viernes?

Lunes, Miercoles y Jueves 7pm

**Reply by ElizaFlores26** [reply]
muchas gracias :D

**Comment by luandogo** [comment, marked as answer]
Ya está disponible! :)

**Comment by Lfer1111** [comment]
Hola a todos! alguien me pudiera ayudar con el link de la transmicion de hoy? gracias =)

**Reply by MaribelMendezM** [reply]
x2

**Reply by Hisahito** [reply]
Ya esta publicado en la Semana 1 de FrontEnd de Teams ! 
Aqui lo dejo: 
https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZTgyN2JhMGYtN2QwNi00NWFmLTk0NzItY2ExY2NkOWMxNTMx%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Reply by MaribelMendezM** [reply]
mil gracias!!!

**Reply by ghost** [reply]
https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZTgyN2JhMGYtN2QwNi00NWFmLTk0NzItY2ExY2NkOWMxNTMx%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d
